src/main/java/eu/dissco/backend/repository/ElasticSearchRepository.java

```diff
@@ -81,35 +88,43 @@ public Pair<Long, List<DigitalSpecimenWrapper>> getLatestSpecimen(int pageNumber
     return getDigitalSpecimenSearchResults(searchRequest);
   }
 
-  public List<AnnotationResponse> getLatestAnnotations(int pageNumber, int pageSize)
+  public List<Annotation> getLatestAnnotations(int pageNumber, int pageSize)
       throws IOException {
     var offset = getOffset(pageNumber, pageSize);
     var pageSizePlusOne = pageSize + ONE_TO_CHECK_NEXT;
     var searchRequest = new SearchRequest.Builder()
         .index(properties.getAnnotationIndex())
-        .sort(s -> s.field(f -> f.field(FIELD_CREATED).order(SortOrder.Desc))).from(offset)
+        .sort(s -> s.field(f -> f.field(FIELD_CREATED_ANNOTATION).order(SortOrder.Desc))).from(offset)
         .size(pageSizePlusOne).build();
     return client.search(searchRequest, ObjectNode.class).hits().hits().stream().map(Hit::source)
         .map(this::mapToAnnotationResponse).toList();
   }
 
-  private AnnotationResponse mapToAnnotationResponse(ObjectNode json) {
-    var annotation = json.get("annotation");
-    var createdOn = parseDate(annotation.get(FIELD_CREATED));
-    var generatedOn = parseDate(annotation.get(FIELD_GENERATED));
-    return new AnnotationResponse(
-        HANDLE_STRING + json.get("id").asText(),
-        json.get("version").asInt(),
-        getText(annotation, "type"),
-        getText(annotation, "motivation"),
-        annotation.get("target"),
-        annotation.get("body"),
-        annotation.get("preferenceScore").asInt(),
-        getText(annotation, "creator"),
-        createdOn,
-        annotation.get("generator"),
-        generatedOn,
-        null);
+  private Annotation mapToAnnotationResponse(ObjectNode annotation) {
+    var createdOn = parseDate(annotation.get(FIELD_CREATED_ANNOTATION));
```

Comment:
Wonder if we cannot just map with the objectMapper directly to Annotation

Reply:
done

Reply:
Sweet that saves a lot of code